RecoTracker/MeasurementDet/plugins/TkGluedMeasurementDet.cc

```diff
@@ -139,7 +152,8 @@ bool TkGluedMeasurementDet::measurements( const TrajectoryStateOnSurface& stateO
 				      (theMonoDet->hasAllGoodChannels() || 
 				       testStrips(stateOnThisDet,gluedPlane,*theMonoDet)
 				       )
-				      ) /*Mono OK*/ || 
+				      ) /*Mono OK*/ 
+                                     && // was || 
```

Comment:
can this "||" <-> "&&" be made configurable as well? 
e.g. use "||" for "minPt2ForHitRecoveryInGluedDet >= 1E10" (100000 = 1E5 is the pt value for the switch to go off)